### PR TITLE
added details for adding a new file to README

### PR DESCRIPTION
**Pull request by tannazie**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,6 +25,23 @@ with the documentation generation system.
...
+
+```
+---
+layout: default
+title: Pipeline Integrations
+pagename: pipeline-integrations
+lang: en
+---
+```
+
```
Comment (willis102) [inline README.md]: My only suggestion is that we could expand on this a little with an info box on special characters.  This has been a sticking point a couple times, and while we have a ticket in to improve the error message / CI status around this, clearer documentation could be beneficial as well.  Maybe something like:

```suggestion

{% include info title="Special Characters in Titles" content="If using special characters (including colons) in page titles, it's necessary to wrap the title in ASCII double quotes." %}
```

**Comment by willis102** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/609350ab1d1101d2addac5e9fc0e8799136fa0fd/index.html)

File: README.md (hunk 1 above)
Comment (tannazie) [reply README.md]: will take your content, but change it to markdown so it actually looks right with github.com. thanks!

**Comment by willis102** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/4a3532566ce65647fc4e0d3100c437402b2a0488/index.html)

**Pull request merged**